sirepo/template/radia_util.py

```diff
@@ -242,10 +277,11 @@ def kick_map(
         g_id, begin, dir_long, num_periods, period_length, dir_trans, range_trans_1,
         num_pts_trans_1, range_trans_2, num_pts_trans_2
     ):
-    return radia.FldFocKickPer(
```

Comment:
Why introduce a temporary variable?

Reply:
When value involves a method call with a lot of parameters I think it's easier to see what's going on with a temp variable.  One fewer pair of parens to navigate.  Maybe it's just me, and I try not to overdo it